config/ppo/FoodCollector.yaml

```diff
@@ -24,3 +24,28 @@ behaviors:
     time_horizon: 64
```

Comment:
Should we delete the FoodCollector one as we don't have that scene anymore? Or maybe we should just rename the behavior to `FoodCollector`

Reply:
Just did this